packages/components/src/components/TableComponent/TableComponent.scss

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.tableComponent {
+  .bx--table-toolbar {
```

Comment:
why are we overriding the Carbon background colours?

Reply:
Only reason was to match others tables in the dashboard and also, IMO it looks better with these color scheme. That being said, I can revert back to original:

<img width="1378" alt="Screen Shot 2019-11-08 at 10 50 42 AM" src="https://user-images.githubusercontent.com/49996607/68489723-a429e800-0215-11ea-8fa7-ff7c25774152.png">